int32 a
---
int32 intg_us_new
